blob/blob.go

```diff
@@ -108,6 +109,29 @@ func (b *Blob) Index() int {
 	return b.index
 }
 
+func (b *Blob) Length() (int, error) {
+	s, err := BlobsToShares(b)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(s) == 0 {
+		return 0, errors.New("incorrect number of shares")
+	}
+
+	appShare, err := shares.NewShare(s[0])
+	if err != nil {
+		return 0, err
+	}
+
+	length, err := appShare.SequenceLen()
+	if err != nil {
+		return 0, err
+	}
+
+	return shares.SparseSharesNeeded(length), nil
```

Comment:
What is this call for and why returning length from SequenceLen is not enough?

Reply:
`SequenceLen` returns the sequence length(and not the exact number of shares). `SparseSharesNeeded` returns the number of shares needed to store the sequenceLen.